### DISCUSSION TITLE
Raspberry PI OS 64-bit: is it working?

### DISCUSSION THREAD

**Discussion by rhpk** [tiny-pilot/tinypilot#563] [General]
Hi, I was able to run this wonderful tool previously with a 32-bit version of Raspbian on a Raspberry PI 4. It worked flawlessly so my kudos to the developers!

Anyhow, for a number of reasons, I needed to switch to 64-bit with Raspberry PI OS (Buster) and OTG HID stopped working. The g_hid kernel module is not even loaded and if I do, I get an error (no such device).

Of course, I've run rpi-update but to no avail.

Anybody with some experience to share with 64-bit version?

Thanks!

**Comment by mtlynch** [comment]
Unfortunately, only the 32-bit version of Raspberry Pi OS is officially supported. There's a [thread here](https://github.com/mtlynch/tinypilot/issues/79) of community members experimenting with different OSes and hardware, some of which are 64-bit.

**Reply by rhpk** [reply]
Thanks for the swift reply. And thanks for this wonderful project!
